proxy/proxy/broker/rhnBroker.py

```diff
@@ -546,6 +544,30 @@ def __cacheClientSessionToken(self, headers):
         self.proxyAuth.set_client_token(self.clientServerId, token)
         return token
 
+
+    def __update_token_channels(self, headers, token=None):
+        log_debug(1)
```

Comment:
Do we need it?

Reply:
there are plenty of those log entries... but I can remove that one for sure